Task_1/fets_challenge/inference.py

```diff
@@ -266,6 +266,9 @@ def model_outputs_to_disc(data_path,
         # convert array to SimpleITK image 
         image = sitk.GetImageFromArray(output)
 
+        # get the image information such as affine orientation from the label
+        image.CopyInformation(sitk.ReadImage(subject['label']['path']))
+
```

Comment:
Would it be better to get this information from the first channel image, instead? This way, this will work during the testing phase, when there are no labels present... 😄 

```suggestion
        # get the image information such as affine orientation from the label
        image.CopyInformation(sitk.ReadImage(subject['0']['path']))

```

Reply:
Thanks Sarthak.